CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -90,14 +132,18 @@ static ResizePoolViewModel()
             deallocationOptionValues.AddRange(Enum.GetNames(typeof (ComputeNodeDeallocationOption)));
         }
 
-        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated)
+        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated, string currentAutoScaleFormula)
         {
             this.batchService = batchService;
 
             this.PoolId = poolId;
             this.TargetDedicated = currentDedicated ?? 0;
-            this.DeallocationOptionString = null;
-
+            this.DeallocationOptionString = null;
+            if (string.IsNullOrEmpty(currentAutoScaleFormula))
+                this.IsFixed = true;

```

Comment:
Add { } around the statement.

I have removed isFixed as it can be managed with one property as well. A person can set isAutoScale property from the checkbox on the UI, so I believe it must be writable.